charts/loki-simple-scalable/values.yaml

```diff
@@ -173,6 +173,17 @@ prometheusRule:
 write:
   # -- Number of replicas for the write
   replicas: 1
+  autoscaling:
+    # -- Enable autoscaling for the write
+    enabled: false
+    # -- Minimum autoscaling replicas for the write
+    minReplicas: 1
```

Comment:
can we make the `minReplicas` 3? One of the goals of simple-scalable is to use Loki defaults as much as possible. I'd like to be able to use the default replication_factor of 3, which would require `minReplicas` to be 3. Furthermore, anyone who has a big enough cluster that they need to enable autoscaling probably also wants a replication factor of 3 so they have failure tolerance.

Reply:
I support minReplicas = 3.